x-pack/plugins/rule_registry/docs/alerts_client/classes/alertsclient.md

```diff
@@ -41,7 +42,7 @@ on alerts as data.
 
 #### Defined in
 
-[rule_registry/server/alert_data_client/alerts_client.ts:59](https://github.com/elastic/kibana/blob/f2a94addc85/x-pack/plugins/rule_registry/server/alert_data_client/alerts_client.ts#L59)
+[rule_registry/server/alert_data_client/alerts_client.ts:67](https://github.com/dhurley14/kibana/blob/cd896fac629/x-pack/plugins/rule_registry/server/alert_data_client/alerts_client.ts#L67)
```

Comment:
These probably should continue pointing to the elastic repo :)

Reply:
Every time I generate the docs I forget to add `--upstream`. Thank you!

Reply:
fixed here: f6e6b921cafe96cd56774d6c8ab65f84d3dafbdc